pkgs/by-name/cw/cwtch/package.nix

```diff
@@ -30,14 +30,15 @@ buildGoModule rec {
 
   buildPhase = ''
     runHook preBuild
-    make linux
+    make libCwtch.so
```

Comment:
I tested and confirmed that you could change this to:
```EXPERIMENTS="serverExperiment" make libCwtch.so```

And that would enable it to host a Server.
Reference: https://git.openprivacy.ca/cwtch.im/autobindings#experiments

Reply:
I think it would be better to let folks who want to run a server enable it as an override. Since this is security-critical software, we should ship as close as possible to upstream.